Gems/Atom/RPI/Code/Tests/Shader/ShaderTests.cpp

```diff
@@ -538,7 +538,7 @@ namespace UnitTest
 
             auto shaderAsset = shader->GetAsset();
             EXPECT_EQ(shader->GetPipelineStateType(), shaderAsset->GetPipelineStateType());
-            EXPECT_EQ(shader->GetShaderResourceGroupLayouts(), shaderAsset->GetShaderResourceGroupLayouts());
+            EXPECT_EQ(shader->GetShaderResourceGroupLayouts().data(), shaderAsset->GetShaderResourceGroupLayouts().data());
```

Comment:
This is a slightly different compare than before - the ends or size need to be checked as well.

Reply:
I'll update it to take the size into account as well.